File: leetcode/arrays/two_sum/cpp/solution_test.cc
#include <cassert>
#include <vector>

#include "solution.h"

int main() {
    {
        auto got = Solve({2, 7, 11, 15}, 9);
        assert((got == std::vector<int>{0, 1}));
    }
    {
        auto got = Solve({1, 2, 3}, 99);
        assert(got.empty());
    }
    return 0;
}

File: leetcode/arrays/two_sum/scala/SolutionTest.scala
object SolutionTest {
  def main(args: Array[String]): Unit = {
    val got = Solution.solve(Array(2,7,11,15), 9)
    assert(got.sameElements(Array(0,1)))
    val got2 = Solution.solve(Array(1,2,3), 99)
    assert(got2.isEmpty)
    println("OK")
  }
}

